### DISCUSSION TITLE
How can I install Pscale CLI?

### DISCUSSION THREAD

**Discussion by mdshohidulislam1486** [planetscale/discussion#191] [Documentation] [answered]
I followed the doc, but could not install it, sadly there is no video on this topic anywhere, everyone just gives directions to install the Pscale CLI, I think it might be very easy but I tried it for almost one day but could not install the CLI.

**Comment by jgreet** [comment, marked as answer]
Hi, there are installation instructions for OSX, Windows and Linux in the ReadMe file:
https://github.com/planetscale/cli#installation

Perhaps you could tell us what operating system you're using, what you tried, and what went wrong?

Thanks!
